### PR TITLE
hardening/117_getIndividualContextEntityAttribute

### PR DESCRIPTION
**Pull request by kzangeli**
GET /v1/contextentities/EID/attributes/attibuteName now uses Standard Op

### CODE REVIEW

File: src/lib/serviceRoutines/postQueryContext.cpp [hunk 1]
```diff
@@ -215,11 +213,21 @@ std::string postQueryContext
     return answer;
   }
 
-  s = xmlTreat(cleanPayload, ciP, &parseData, RtQueryContextResponse, "queryContextResponse", NULL, &errorMsg);
+  //
+  // NOTE
```
Comment (fgalan) [inline src/lib/serviceRoutines/postQueryContext.cpp]: The same NOTE is include in other GET operations in which the same comment makes sense? Just to maintain "comment coherence" along code...

Comment (kzangeli) [inline src/lib/serviceRoutines/postQueryContext.cpp]: I haven't needed this before, this is the first time.
However, all GET conv-op (that translate into a ngsi10 query) will get to this std-op, so, this unique comment covers them all ...

If the same thing is needed for some other "conv-op translation to std-op", I will try to add the same 'NOTE'.

Comment (fgalan) [inline src/lib/serviceRoutines/postQueryContext.cpp]: Good point. I didn't realice that the comment was in the "common" postQueryContext logic. I though that I was in a per-convop specific part.

I don't like too much manipulating ciP as I see it as an object representing the "connection environment", but I think it is the most pragmatic way of doing. However, maybe I'm not objective... you know I don't like too much ciP ;)

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**